ArduPlane/Plane.h

```diff
@@ -1241,8 +1245,10 @@ class Plane : public AP_Vehicle {
     void failsafe_check(void);
     bool is_landing() const override;
     bool is_taking_off() const override;
-#if AP_SCRIPTING_ENABLED
+#if AP_EXTERNAL_CONTROL_ENABLED
```

Comment:
should be
#if AP_SCRIPTING_ENABLED || AP_EXTERNAL_CONTROL_ENABLED

Reply:
Fixed